parastell/magnet_coils.py

```diff
@@ -609,6 +609,8 @@ def generate_magnet_set():
 
     magnet_coils_dict = read_yaml_config(args.filename)
 
+    logger = log.check_init(None, null_logger=False)
```

Comment:
Similar comment to invessel build - let's actively make a logger if we want one rather than rely on this default approach